src/docfx/build/dependency/Resolve.cs

```diff
@@ -79,24 +81,33 @@ public static (Error error, string content, Document file) TryResolveContent(thi
             return file != null ? (error, file.ReadText(), file) : default;
         }
 
-        public static (Error error, string href, string fragment, Document file) TryResolveHref(this Document relativeTo, string href, Document resultRelativeTo)
+        public static (List<Error> errors, string href, string fragment, Document file) TryResolveHref(this Document relativeTo, string href, Document resultRelativeTo, XrefMap xrefMap = null, DependencyMapBuilder dependencyMapBuilder = null)
         {
             Debug.Assert(resultRelativeTo != null);
+            var errors = new List<Error>();
+
+            if (href.StartsWith("xref:"))
+            {
+                var (uidError, uidHref, _) = TryResolveXref(href.Substring("xref:".Length), xrefMap, dependencyMapBuilder, relativeTo);
```

Comment:
is one `error` enough?

Reply:
Here the only error we need to take care of is `uid-not-found`